### DISCUSSION TITLE
Product price update

### DISCUSSION THREAD

**Discussion by judev-jbg** [saleweaver/python-amazon-sp-api#649] [FAQ] [answered]
Hi, I want to update the price of the products I have in my inventory but I am not clear which endpoint I should use, can you help me please?

**Comment by denisneuf** [comment, marked as answer]
try this for a single product by sku:

from sp_api.api import ListingsItems

        res = ListingsItems(account=store, marketplace=marketplace).patch_listings_item(
            sellerId=seller_id,
            sku=sku,
            marketplaceIds=marketplace_id,
            issueLocale=issue_locale,
            body=dictionary
        )

While your dictionary should look like this:

{'productType': 'PRODUCT', 'patches': [{'op': 'replace', 'path': '/attributes/purchasable_offer', 'value': [{'marketplace_id': 'A1RKKUPIHCS9HS', 'currency': 'EUR', 'our_price': [{'schedule': [{'value_with_tax': 2359.0}]}]}]}]}

**Comment by judev-jbg** [comment]
Thanks for your help @denisneuf  is what I was looking for.
